userspace/libsinsp/filter.cpp

```diff
@@ -344,7 +352,7 @@ Json::Value sinsp_filter_check::rawval_to_json(uint8_t* rawval, const filterchec
 			{
 		 		return (Json::Value::Int64)*(int64_t *)rawval;
 			}
-			else 
+			else
```

Comment:
OK, I was trying to go through and fix spacing issues after our last discussion on tab vs spaces.